zio-http/src/test/scala/zio/http/api/package.scala

```diff
@@ -0,0 +1,7 @@
+package zio.http
+
+import scala.language.implicitConversions
+
+package object api {
+  implicit def stringToIn(s: String): In[Unit] = In.literal(s)
```

Comment:
Let's put this into the companion object of `In`, where it will be searched in the event a `String` is encountered but an `In` is expected.